test/functional/apps/settings/_index_pattern_popularity.js

```diff
@@ -48,9 +47,11 @@ define(function (require) {
             return common.sleep(1000);
           })
           .then(function openControlsByName() {
-            return settingsPage.openControlsByName(fieldName);
+            common.debug('Starting openControlsByName "geo.coordinates"');
+            return settingsPage.openControlsByName('geo.coordinates');
```

Comment:
fieldName variable was used in a couple of other places and should be used here too instead of the hard-coded value.  Fixed.